src/Filter/modules/DurationFilter/DurationFilter.tsx

```diff
@@ -0,0 +1,71 @@
+import { FilterModule } from 'src/Filter/types';
+import MinMaxInput, { MinMax } from './MinMaxInput';
+
+export const getMinMaxFromKey = (key: any): MinMax | undefined => {
```

Comment:
I have moved the function to the util file and added test cases. Also, I tried with string instead of any. But there was lot of conversion that needs to be done to return the appropriate values so I am using 'any'.